tools/site_and_regional/run_neon.py

```diff
@@ -508,14 +525,14 @@ def run_case(
             with Case(base_case_root, read_only=False) as basecase:
                 print("---- cloning the base case in {}".format(case_root))
                 basecase.create_clone(
-                    case_root, keepexe=True, user_mods_dirs=user_mods_dirs
+                    case_root, keepexe=True
```

Comment:
do we need to keep this so user_mod_dirs are set up correctly when cloning a new case?

Reply:
@ekluzek I've added this back into run_neon so cloned cases point to the correct DATM inputs.  Does it mess up other aspects of the work flow you'd modified?

Reply:
OK, good to know. By removing it the duplication in user_nl_* files was removed. But, it sounds like there are two side-effects of removing it. One is that the shell_commands file is removed, which I figured was maybe OK. But, the other is apparently what you pointed out that it's needed so that the DATM inputs are handled.

So I'll add a comment that clarifies that keeping this around is important.

Reply:
OK, I added a comment